app/views/recipes/public_recipes.html.erb

```diff
@@ -0,0 +1,56 @@
+<% if @public_recipes.empty? %>
+  <div class='text-center'>
+    <h1 class="text-3xl font-bold my-8">Hello! We don't have any recipes yet!</h1>
+    <p class='mb-5'>Please be the first to create one!</p>
+    <% if current_user %>
+      <%= link_to 'Create a recipe', new_recipe_path, class: 'bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded' %>
+    <% else %>
+      <%= link_to 'Login', new_user_session_path, class: 'bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded' %>
+    <% end %>
+  </div>
+<% else %>
+  <h1 class="text-3xl font-bold mb-4 text-center mt-8 pb-2 border-b border-gray-300">Public Recipes</h1>
```

Comment:
- [ ] Good job so far but I have an issue with your design. I noticed that your public recipe list doesn't follow the project wireframe. Please fix it in order to make your app design follows the wireframe 👍 

#### Your design
![public_recip](https://github.com/Solobarine/recipes/assets/6615394/4ed3e447-1211-46a1-a009-284f0f92a871)

#### Wireframe
![public_recipe_index](https://github.com/Solobarine/recipes/assets/6615394/c0425807-42f7-47b3-b53b-e44da24043ce)